osu.Server.Queues.ScoreStatisticsProcessor/Processors/ScorePerformanceProcessor.cs

```diff
@@ -163,13 +163,27 @@ public static bool AllModsValidForPerformance(Mod[] mods)
                     case ManiaModKey10:
                         return false;
 
+                    case ModNightcore nc:
```

Comment:
wonder if it's worth casing `ModRateAdjust` at the end of the switch to cover all of these? rather than having individual handling?

Reply:
Doesn't it feel iffy to blindly allow all rate adjust mods?

Reply:
Judging that we're in this predicament due to relying on inheritance hierarchies, I think specifying mod classes explicitly is better.

You could _maybe_ try to keep one unified copy of the code under four case labels. Maybe.

Reply:
> You could maybe try to keep one unified copy of the code under four case labels. Maybe.

Might read better, not fussed either way just curious on preference.